internal/cache/profiles.go

```diff
@@ -29,36 +25,33 @@ type ProfileCache interface {
 	Update(profile models.DeviceProfile) errors.EdgeX
 	RemoveByName(name string) errors.EdgeX
 	DeviceResource(profileName string, resourceName string) (models.DeviceResource, bool)
-	CommandExists(profileName string, cmd string, method string) (bool, errors.EdgeX)
-	ResourceOperations(profileName string, cmd string, method string) ([]models.ResourceOperation, errors.EdgeX)
-	ResourceOperation(profileName string, deviceResource string, method string) (models.ResourceOperation, errors.EdgeX)
+	DeviceCommand(profileName string, commandName string) (models.DeviceCommand, bool)
+	CommandExists(profileName string, cmd string) (bool, errors.EdgeX)
```

Comment:
`CommandExists` could be replaced by `DeviceCommand(profileName string, commandName string) (models.DeviceCommand, bool)`